frontend/src/metabase/dashboard/containers/DashboardHeader.jsx

```diff
@@ -459,10 +461,15 @@ class DashboardHeader extends Component {
     return buttons;
   }
 
-  saveAsImage = async () => {
+  saveAsPDF = async () => {
     const { dashboard } = this.props;
     const cardNodeSelector = "#Dashboard-Cards-Container";
-    await saveDashboardPdf(cardNodeSelector, dashboard.name);
+    await saveDashboardPdf(cardNodeSelector, dashboard.name).then(() => {
+      trackSchemaEvent("dashboard", "1-0-2", {
```

Comment:
Could you add another `analytics.ts` file in `dashboard` and move `trackSchemaEvent` call there?